automation/addLabelsOnPullRequest.automation.js

```diff
@@ -18,42 +18,40 @@ exports.addApprovedLabel = async context => {
 
 	const reviews = await context.octokit.pulls.listReviews(params);
 	const approvedLabel = issueLabels.data.filter(
-		label => label.name === ':white_check_mark: Approved'
+		label => label.name === ":white_check_mark: Approved"
 	);
 	const changesRequestedLabel = issueLabels.data.filter(
-		label => label.name === ':warning: Changes requested'
+		label => label.name === ":warning: Changes requested"
 	);
 
 	const approvedReviews = reviews.data.filter(
-		review => review.state === 'APPROVED'
+		review => review.state === "APPROVED"
 	);
 
 	const foundChangesRequestedReview = issueLabels.data.filter(
-		review => review.state === 'CHANGES_REQUESTED'
+		review => review.state === "CHANGES_REQUESTED"
 	);
 
 	if (
 		foundChangesRequestedReview.length === 0 &&
 		changesRequestedLabel.length > 0
 	) {
-		removeLabel([':warning: Changes requested'], context);
-	}
-
-	if (approvedReviews.length === 0 && approvedLabel.length > 0) {
-		removeLabel([':white_check_mark: Approved'], context);
+		removeLabel([":warning: Changes requested"], context);
 	}
 
 	if (approvedReviews.length > 0) {
 		const issueLabels = await listIssueLabels(context);
 
 		const foundReviewLabel = issueLabels.data.filter(
-			label => label.name === ':mag: Ready for Review'
+			label => label.name === ":mag: Ready for Review"
 		);
 
-		addLabel([':white_check_mark: Approved'], context);
+		if (approvedLabel.length === 0) {
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹` with `····`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
    if (approvedLabel.length === 0) {
```